content/_releases/22/44.md

```diff
@@ -0,0 +1,95 @@
+title: Product Update - v22.44
+description: Details of the product update v22.44 from 3rd of November 2022.
+layout: rel
+version: "22.44"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-11-03
+redirect_from:
+  - /releases/22.44.html
+---
+
+## Improvements
+
+### Component repositories: New design & improved functionality
+
+We continue to update our User Interface to improve ease of use. This time it is
+the UI of every component repository that is updated. When you navigate to
+Development > Developer Teams > Random component your team you will notice new
+uniform design:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2244/RN2244-repository-view.png" title="Component repositories: New design Inside" %}
+
+From each component repository page you can navigate to the environment variables
+page where you can add new, change or delete the existing environment variables
+for the component:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2244/RN2244-repository-env-vars.png" title="Component repositories: environment variables" %}
+
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### RabbitMQ HAProxy in kubernetes cluster
+
+The HAProxy for RabbitMQ cluster will now run as a separate service in the platform
+kubernetes cluster instead of dedicated single virtual machine. This enables
+high-availability and scalability by running the service in a replica set.
+
+To run the HAProxy for the RabbitMQ cluster, change the following parameters in
+the HELM3 charts:
+
+*   Update all HELM3 variables pointing to HAProxy VM to `haproxy-service.platform.svc.cluster.local`.
+*   Introduce a new HELM3 configuration file `haproxy.yaml` with the following content:
+```yaml
+haproxy:
+      rabbitmqNodeIps:
+        rabbitmq: "10.28.0.4"
+        rabbitmq-2: "10.28.0.65"
+```
+Where `rabbitmq` and `rabbitmq-2` parameters must match the RabbitMQ cluster VM
+names and values to their VM IP addresses.
+
+> **Please Note**: You must first deploy the HAProxy chart before the platform version deployment.
+
+
+{% endif %}
+
+
+## Fixed bugs
+
+*   Fixed the user interface bug when flow status changes were not reflected on the flows page.
+*   Addressed the problem when removing the user from the workspace or contract would take more than 4 minutes. Now the process takes less than a second.
+*   Fixed the bug when API would return `5xx` error when you use a space in the repository name while creating a component repository. Now the UI will disable the save button and show en error message `Repository name contains invalid characters. Letters, digits, - and _ are allowed only`
+*   Addressed the bug when identity keys page (SSH Keys) would not load in case when a key without title were saved previously. Now the title is a required field:
```

Comment:
...would not load when a key was previously saved without a title. The title is now a required field.